### ISSUE TITLE
Attenuation hides tileset at first activation

### ISSUE THREAD

**Issue by SunBlack** [CesiumGS/cesium#9993]
Following codes hides a tileset on first call:
```
tileset.pointCloudShading.attenuation = true;
viewer.scene.requestRender();
```

When it will be enabled a second time, it works:

![Attenuation](https://user-images.githubusercontent.com/483922/148369742-aa6eaa39-1fab-4e50-bcb1-ae04ea4da30b.gif)

Sandcastle example: [Link](https://sandcastle.cesium.com/#c=jVPBUtswEP2VnZycmVSxYzl2QmCahkOZgYGBtL34osibRK0sUUmGph3+vbJs2lA49GTr7du32t0nrpV18CDwEQ2cgsJHWKEVTU0+BywqBzycV1o5JhSacjCCX6UCcGiMR+bPCevuTLZG11+0kVUPRMNRS7ccFabn10oeYA7ONBhgJ2qUXncOWyZthzElauaEVnCMGvzeoHW3qCo0V7rCY5ma/RB1U3fBtddc7ZnatZQLtRVKuEOpnoYnpSrVeAw3WigHK6mbCjYHuDF6S+AGfUewlBLVCFZaNvVGMPikxAMa6wXgVm+0E9zCJdsQuONMKaF2rcBS4lemKqNhbTTnWgrfQwVXzKGAldCcGSmQlIqHcTsh0aLz82aPTLjnAXaf9HzdhcMgL7RaWn+4qKJkSidJ20K3LRIGSu6NqIXzd7SEVVXUS3edviByVqNhxAfbzUZhhZWfp1Bh1PPj3a+Ycf6PqTRqaQCUxklOKZlSmtAsTeiowyeTLM4SQpM4L2ZJEmeTPkCzWUYnMUlzmheUpmnAOydoI9Cb6VXVj8gqP9Ab4fj+VkvZ18581WmepkWcxeksz/K+xLuYTPLpNM6TLC3SWVFM+8CUTIrEX6bIMxrHcZH9qd074M4vhzPrJLYzW+vdTuKHxjmtQslysHQOVRNu6O3e2bTbyX3rnGCcu324LWF/uYG5bRQP3o34Hvk3rIbdc/lPDW+KPu+ky3qxxBdvIBoGylOphieD0WBh3UHiWZcF8F7U99o4aIyMCBk7rO+l96Mdbxqv7gi3dtiXWIyPUxeVeABRnb7x8oFLZq2PbBsp78RPLAdni7Hnv0qVOnR27d+OZIeWtk/OLjuQELIY++PbmU5ruWHmH+Xf)

Browser: Opera 82

Operating System: Win10

**Comment by lilleyse** [comment]
I can reproduce on Ubuntu 21 and Chrome.

**Comment by ggetz** [comment]
I can reproduce on Windows 10 and Chrome, but not on MacOS and Chrome.

**Comment by sanjeetsuhag** [comment]
I only see this on Chrome 97.0.4692.99 on Windows 10. Firefox 96.0.2 worked fine

**Comment by wonszopaionk** [comment]
@lilleyse @ggetz  Any updates on this? Or was it fixed already? I'm on cesium 1.87

**Comment by SunBlack** [comment]
@wonszopaionk In my initial post is a link to a sandcastle demo, there you can try it yourself. And no, it is not fixed until now.

**Comment by wonszopaionk** [comment]
@SunBlack yeah, my bad - it worked for me on firefox within this sandcastle, but problem still exists on other browsers.
@lilleyse @ggetz any updates?

**Comment by ggetz** [comment]
No updates at the moment. If you can find any further information which could help us track down the cause, we'd appreciate it!

**Comment by SunBlack** [comment]
Updated Sandcastle example. We have the issue still with Chrome 117 + Cesium 1.127

**Comment by SunBlack** [comment]
Seems to be fixed with 1.144.0. Maybe due to #13662.
